### PR TITLE
Upgrade to ISPN 15.0.11.Final

### LINKED ISSUE #34931: Upgrade to ISPN 15.0.11.Final

### PR DESCRIPTION
**Pull request by pruivo**
Closes #34931

### CODE REVIEW

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for the PR. I suppose one more version needs to be updated in-line with the Infinispan version. Please have a look. Thanks!

File: pom.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
         <h2.version>2.3.230</h2.version>
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
-        <infinispan.version>15.0.10.Final</infinispan.version>
+        <infinispan.version>15.0.11.Final</infinispan.version>
         <!-- version number is needed for the proto-schema-compatibility-maven-plugin and needs to be in-sync with the version used in Infinispan -->
         <protostream.version>5.0.8.Final</protostream.version>
```
[the remark this answers is not in the document]
Comment (pruivo) [reply pom.xml]: Infinispan 15.0.10.Final already uses ProtoStream 5.0.10.Final. I've removed this property.

**Review by ahus1 — APPROVED** [review]
Thank you for the change!

**Comment by ahus1** [comment]
fyi: @ryanemerson

**Pull request merged**